dnpchallenge/iedconfig.xml

```diff
@@ -37,16 +37,20 @@
   </dnp3>
   <logic>
     <period>1s</period>
+    <process-updates>true</process-updates>
```

Comment:
Because the ot-sim RTU device can send DNP3 commands to this IED device that the logic module needs to process, the logic module must be configured to listen for update messages from the message bus. By default this is disabled.